src/components/button-toggle/button-toggle-icon.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {
+  StyledButtonToggleIcon,
+  StyledButtonToggleIconProps,
+} from "./button-toggle.style";
+import Icon, { IconType } from "../icon";
+
+export interface ButtonToggleIconProps extends StyledButtonToggleIconProps {
+  /**
+   * <a href="https://brand.sage.com/d/NdbrveWvNheA/foundations#/icons/icons" target="_blank">List of supported icons</a>
+   *
+   * buttonIcon to render.
+   */
+  buttonIcon: IconType;
+  /** Sets the icon in the disabled state */
+  disabled?: boolean;
+}
+
+const ButtonToggleIcon = ({
+  buttonIcon,
+  buttonIconSize,
+  disabled,
```

Comment:
```suggestion
const ButtonToggleIcon = ({
  buttonIcon,
  buttonIconSize,
  disabled,
  hasContent,
```
I think we need to surface `hasContent` as a prop, otherwise we could make the `margin-right` css rule for `StyledBUttonToggleIcon` default behaviour?